packages/flutter/test/widgets/navigator_test.dart

```diff
@@ -379,6 +380,49 @@ void main() {
     expect(isPopped, isTrue);
   });
 
+  testWidgets('Push and Pop should send System Channels messages', (WidgetTester tester) async {
```

Comment:
here and everywhere else in this CL, replace "System Channels message" with "platform message" which is the more commonly used term

Reply:
Done.